### PR TITLE
docs : add animations to readme file

### PR DESCRIPTION
**Pull request by Piyush132000**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

Updated the readme file for adding one new topic in Angular Advance topics. 
Added animations in listing of advance topics of  Angular.

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
same

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Get started with Angular, learn the fundamentals and explore advanced topics on
 - [Server Side Rendering][ssr]
 - [Schematics][schematics]
 - [Lazy Loading][lazyloading]
+- [Animation][animation]
```
Comment (devversion) [inline README.md:56]:
```suggestion
- [Animations][animations]
```

File: README.md [hunk 2]
```diff
@@ -157,3 +158,4 @@ Join the conversation and help the community.
 [stackoverflow]: https://stackoverflow.com/questions/tagged/angular
 [youtube]: https://youtube.com/angular
 [meetup]: https://www.meetup.com/find/?keywords=angular
+[animation]:https://angular.io/guide/animations
```
Comment (devversion) [inline README.md:161]:
```suggestion
[animations]: https://angular.io/guide/animations
```

**Review by devversion — COMMENTED** [review]
Thank you for the PR. I'm not sure if this is something we want to add here, so I'm adding @mgechev, in case you have thoughts on that.

I've requested two small changes on the PR, and it would also be great if you could have a look at fixing the commit message. See [the contribution guide/doc](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)

File: README.md (hunk 1 above)
Comment (Piyush132000) [reply README.md]: thanks for your feedback

**Comment by Piyush132000** [comment]
I have done changes suggested by @devversion.

**Comment by dylhunn** [comment]
@Piyush132000 You'll also need to squash all the commits into one, and update the message format:

```
docs: add animations to readme file
  
<add long description here>
```

**Review by devversion — APPROVED** [review]
LGTM, but the commits need to be squased and named according to @dylhunn's message. Thank you!

**Comment by Piyush132000** [comment]
Hlo everyone , sorry for inconvenience , i am  trying to squashing my commits and mash happen.
I am closing this PR and raising new PR with proper commit and description.

Sorry for inconvenience.

**Comment by Piyush132000** [comment]
Hlo everyone ,sorry for inconvenience , I raised PR  again please review it.

https://github.com/angular/angular/pull/45488

**Pull request closed without merging**